airflow/serialization/enums.py

```diff
@@ -48,3 +48,4 @@ class DagAttributeTypes(str, Enum):
     TASK_GROUP = 'taskgroup'
     EDGE_INFO = 'edgeinfo'
     PARAM = 'param'
+    XCOM_REF = 'xcom_ref'
```

Comment:
nit -- no strong opinion

```suggestion
    XCOM_REF = 'xcomref'
```

to have same convention as the other enum items

Reply:
Good point -- better to be consistent.